src/main/kotlin/step4/racingcar/racingcar/CarRacing.kt

```diff
@@ -0,0 +1,22 @@
+package step4.racingcar.racingcar
+
+class CarRacing(racers: List<String>, private val moveCount: Int) {
```

Comment:
```suggestion
class CarRacing(cars: List<Car>, private val moveCount: Int) {
```
CarRacing에는 자동차를 탄 레이서가 있어야 된다고 생각해요! 😊

List<Car>로 바꾸면, 어떻게 String 배열의 racers를 넣어야 할까요?

Reply:
CarRacing 클래스에서 레이서를 받는 방식이 아닌 차량을 받도록 수정하겠습니다~